src/components/app.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import axios from 'axios';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+
+const url = 'https://mcr-codes-weather.herokuapp.com/forecast';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+      errorMessage: '',
+      searchText: '',
+    };
+  }
+
+
+  componentDidMount() {
+    this.updateCity('Hebden Bridge');
+  }
+
+  updateCity = (city) => {
+    this.setState({ errorMessage: '' });
+    axios.get(`${url}?city=${city}`)
+      .then(response => {
+        this.setState({
+          forecasts: response.data.forecasts,
+          location: response.data.location,
+        });
+      })
+      .catch(err => {
+        this.setState({
+          errorMessage: 'There are no results, try again',
+        });
+      });
+  };
+
+  handleKeyDown = (event) => {
+    if (event.keyCode === 13) {
```

Comment:
Try to avoid using `magic numbers`. Adding a `const ENTER_KEY = 13` to your top scope would be a lot more informative for other people reading your code.